sli/interpret.cc

```diff
@@ -943,7 +946,8 @@ SLIInterpreter::message( std::ostream& out,
       if ( i != 0 && text_str.at( i - 1 ) == ' '
         && static_cast< int >( space - i ) > static_cast< int >( width - pos ) )
       {
-        out << std::endl << std::string( indent, ' ' );
+        out << std::endl
+            << std::string( indent, ' ' );
         pos = 0;
       }
 
```

Comment:
This file has nothing to do with the pull request and just got in due to clang reformatting. Please remove from PR.